src/SecretsApi/SecretsApi.php

```diff
@@ -22,11 +22,24 @@ class SecretsApi
     private function getBaseURI()
     {
         $config = $this->request()->getConfig();
+
+        $protocol = $config->get('papi_protocol') ?? $config->get('protocol');
+        $port = $config->get('papi_port') ?? $config->get('port');
+        $host = $config->get('papi_host');
+        if (!$host && strpos($config->get('host'), 'hermes.sandbox-') !== false) {
+            $host = 'pantheonapi.sandbox-eco.sbx01.pantheon.io';
```

Comment:
This shouldn't be hardcoded to `sandbox-eco`; take it from `hermes.sandbox-NAME` in the config `host` variable.

Reply:
That makes sense. Fixed